### ISSUE TITLE
failAfterError is not triggered in 0.4.0

### ISSUE THREAD

**Issue by message** [adametry/gulp-eslint#30]
In `0.2.2` works fine. Not quite sure where is the problem.

```js
var gulp = require("gulp");
var eslint = require("gulp-eslint");
var config = require("../config.js");

var isDevelopmentEnvironment = (process.env.NODE_ENV === "development");

gulp.task("lint", function() {
	var sourceJs = [config.source, config.browserify.javascriptsFolder].join("/");
	gulp.src([sourceJs + '/**/*.js'])
		.pipe(eslint({useEslintrc: true}))
		.pipe(eslint.format())
		.pipe(eslint.failAfterError())
		.on("error", function() {
			this.emit("end"); // Keep gulp from hanging on this task
			if (!isDevelopmentEnvironment) {
				console.log('EXIIIIIIIIIIIIIIIIIITTTTTTTTTTT');
				process.exit(1);
			} // Exit with error when in production
		}
	);
});
```

**Comment by shinnn** [comment]
Return the `gulp.src` stream.

```js
return gulp.src([sourceJs + '/**/*.js'])
  .pipe(eslint({useEslintrc: true}))
```

Isn't it fix the problem?

**Comment by message** [comment]
Yes, you're right. Sorry.

**Comment by shinnn** [comment]
No problem!
